builder/test-integration/spec/hypriotos-image/device-init_spec.rb

```diff
@@ -1,28 +1,9 @@
 require 'spec_helper'
 
-# describe package('device-init') do
-#   it { should be_installed }
-# end
-
-# describe command('dpkg -l device-init') do
-#   its(:stdout) { should match /ii  device-init/ }
-#   its(:stdout) { should match /0.1.8/ }
-#   its(:exit_status) { should eq 0 }
-# end
-
 describe file('/boot/device-init.yaml') do
```

Comment:
Better remove this test for file '/boot/device-init.yaml' when the file is removed from the image.